spec/ci_ruby_version_spec.rb

```diff
@@ -9,9 +9,9 @@
   pages_versions = JSON.parse(open('https://pages.github.com/versions.json').read)
   pages_ruby_version = pages_versions['ruby']
 
-  ci_config_file = '.travis.yml'
+  ci_config_file = '.github/workflows/test.yml'
   ci_config = YAML.load_file(ci_config_file)
-  ci_ruby_version = ci_config['rvm'][0]
+  ci_ruby_version = ci_config['jobs']['test']['steps'][1]['with']['ruby-version']
```

Comment:
This may be better handled by setting an environmental variable higher up, or by someone that gets Yaml parsing in Ruby better than me